_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+2. **Automating compliance, testing, integration and deployment**: I mentioned this in the previous point but it is important I highlight this requirement specifically. DevOps relies on automating processes. In a modern technology organization, there is simply too much to be able to manually check in a timely manner with realistic resources. IT operational staff are shifted from the reactive role of putting out fires and instead are responsible for developing and improving tooling and pipelines that ensure compliance with every automated build and during development. For example, Enterprise Architecture would be responsible for building, developing and improving reusable components that, when used by a developer, will automatically implement reference architecture in applications. They are also responsible on mentoring and teaching developers through their wealth of knowledge on creating modular and decoupled applications.
+
+3. **Enabling a continuous improvement environment**: As technologies and requirements change rapidly over the course of time, the only solution is to be proactive and not reactive. As well as rapid changes, it is important to accept the complexity of making technology work together. Very rarely will we be able to get any service or application architecture exactly right and it is difficult to realize this until an application is actually in production. A significant amount of time should be devoted every day to see how processes and technology can be improved to alleviate bottlenecks. If you're wondering how this is possible when you are so overburdened with your current work, strategy #2 ( automation ) is supposed to alleviate this so that you can focus on this valuable work.
+
+In a DevOps release process, 2 weeks of work takes a day or less to release. All the functions are encapsulated within one cross-functional team per product. Interaction with central enablement teams such as IT Security, Accessibility and Testing Services is through automated checks and processes incorporated into the build. These central operational teams are instead tasked with continually addressing bottlenecks and championing improvements rather than solely a fire fighting role. An incident response role still exists, things will go wrong sometimes regardless of any process or methodology taken. The outcome however, is not to add an unrealistic burden to operational staff to manually ensure something doesn't happen again. Problems are fixed at the root by improving the process and incorporating automated checks into build and development tools.
```

Comment:
add an 'as'? to 'An incident response role still exists as things will go wrong sometimes regardless....'
Otherwise I think the comma after 'exists' needs to be a semicolon?

In 'The outcome however'? do we mean outcome? or solution?  Is it the outcome of the incident response role?  if yes, would it be more clear as 'However, the outcome, should not add an unrealistic burden to manually...'

Reply:
The way that government responds to mistakes or stuff slipping through the cracks is to add more governance and manual compliance. So the outcome of mistakes ( incidents ) should not to do this and over burden operational staff but rather add an automated check

Reply:
altered so that the outcome line is more clear